style.css

```diff
@@ -177,6 +181,93 @@
     cursor: pointer;
   }
 
+  .pop-up {
+    background-color: #c1c7d0;
+    padding: 10% 3%;
+    height: 100vh;
+    width: 100vw;
+    position: fixed;
+    overflow-y: scroll;
+    bottom: 0;
+    top: 0;
+  }
+
+  .pop-up-container {
+    background-color: white;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    align-items: center;
+    padding: 3%;
+    border-radius: 20px;
+  }
+
+  .pop-up-head {
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+  }
+
+  .pop-up-title {
+    margin-right: auto;
+  }
+
+  .disabled-button {
+    margin-left: auto;
+    margin-top: auto;
+    margin-bottom: auto;
+    cursor: pointer;
+  }
+
+  .pop-up-canopy {
+    width: 100%;
+    display: flex;
+    justify-content: flex-start;
+  }
+
+  .canopy-items li {
+    display: inline;
+  }
+
+  .pop-up-image {
+    width: 100%;
+  }

```

Comment:
- [ ] [REQUIRED] Good job so far 👍 I really appreciate what you did, but when the `pop-up` window opens, it doesn't fully reflect the elements there, because of the `size` of the pop-up window and the size of the `image` there, and to see the elements that are not fully reflected, you have to scroll more. -down, which may cause your project to be of not good quality. By the way, the `background` value of the pop-up window should be `transparent`, just like `figma design` and the elements below the pop-up window appear to be flattened to the `right`. Please `set` the size of the pop-up window and the elements there as shown in [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1143&t=0rGlfHjhjIsqIf9G-0) 🙏🙏🙏

###  `[HINT]`
```css
/* Container  of pop-up window */
position: absolute;
left: 0px;
right: 0px;
top: 0px;
bottom: 0px;

/* White */
background: #FFFFFF;

/* Neutrals/N30 - L */
border: 1px solid #EBECF0;

/* Base/ Soft */
box-shadow: 0px 48px 48px rgba(37, 47, 137, 0.08);
border-radius: 16px;

/* ------------------------------------------------------------------------------------------------------- */

/* Snapshoot(picture of pop-up) */
position: absolute;
width: 1108px;
height: 586px;
left: 0px;
top: 0px;
``` 
- ### Your design ❌ 

![image](https://user-images.githubusercontent.com/63915024/202268245-c18167ae-df11-4f7e-a205-cfa486c2cac0.png)

- ### [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1143&t=0rGlfHjhjIsqIf9G-0) ✔️ 

![image](https://user-images.githubusercontent.com/63915024/202268203-dd8a9424-7132-4211-bfcd-942335c7a1f5.png)